### PR TITLE
sign_up_window

### PR DESCRIPTION
**Pull request by p0u74**
A series of changes were made :

- Directory added for icon files
- Some variable names changed
- Improvement clean code

### CODE REVIEW

File: sign_up_window.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# sign_up_window
+
+import sys
+
+from PyQt6.QtWidgets import QApplication, QWidget, QLabel, QLineEdit, QVBoxLayout
+from PyQt6.QtGui import QIcon, QFont, QPixmap
+
+app = QApplication(sys.argv)
+class Window(QWidget) :
```
Comment (alirzaMhd) [inline sign_up_window.py:8]: SignUpWindow

File: sign_up_window.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# sign_up_window
+
+import sys
+
+from PyQt6.QtWidgets import QApplication, QWidget, QLabel, QLineEdit, QVBoxLayout
+from PyQt6.QtGui import QIcon, QFont, QPixmap
+
+app = QApplication(sys.argv)
```
Comment (alirzaMhd) [inline sign_up_window.py]: bring that in main!!!!!!!

File: sign_up_window.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        image_box.setGeometry(100, 100, 300, 300)
+        window_layout.addWidget(image_box)
+
+        input_box = QLineEdit(self)
+        input_box.setGeometry(150, 400, 200, 30)
+        window_layout.addWidget(input_box)
+
+        self.setLayout(window_layout)
+
+Window = Window()
```
Comment (alirzaMhd) [inline sign_up_window.py]: bring these in main!!!!!!!

**Comment by alirzaMhd** [comment]
make a main.py file and run those mentioned line in there...
next time dont overdo by removing a branch fix issues in same branch and open new pr

**Comment by alirzaMhd** [comment]
1. Basic import

The simplest way to import a module or package is to use the import keyword. For example, if you have a file called my_module.py, you can import it like this:

import my_module
This makes the code and functions defined in my_module.py available in your current script.

2. Importing specific functions or classes

You can use the from ... import ... statement to import specific functions or classes from a module. For example, if my_module.py contains a function called my_function, you can import it like this:

from my_module import my_function
This allows you to use my_function directly in your script without needing to reference the module name.

3. Importing with an alias

You can use the as keyword to give an imported module or function an alias. This is useful when you want to avoid naming conflicts or make the imported names shorter and more convenient. For example:

import my_module as mm
from my_module import my_function as mf
Now you can use mm and mf as aliases for my_module and my_function, respectively.

4. Importing resources within packages using importlib.resources

importlib.resources allows you to access resources (such as files) within packages. This provides a more consistent way of dealing with files inside your packages and gives you easier access to resource files in other packages. For example, if you have a package structure like this:

books/
│
├── __init__.py
├── alice_in_wonderland.png
└── alice_in_wonderland.txt
You can use open_text() and open_binary() to open text and binary files, respectively:

from importlib import resources

with resources.open_text("books", "alice_in_wonderland.txt") as fid:
    alice = fid.readlines()

with resources.open_binary("books", "alice_in_wonderland.png") as fid:
    cover = fid.read()
[realpython.com](https://realpython.com/python-import/)

5. Adding a directory to sys.path

You can add a directory to the Python import path by modifying the sys.path list. This allows you to import modules from that directory. For example:

import sys
sys.path.append('PATH_TO_DIRECTORY')
Keep in mind that modifying sys.path can potentially cause issues if not done carefully.

[stackoverflow.com](https://stackoverflow.com/questions/15681366/importing-files-in-python)

6. Using __init__.py to make a directory behave like a module

By adding an __init__.py file (which can be empty) to a directory, you can make that directory behave like a module, allowing you to import files from it. This is a better approach than modifying sys.path.

[stackoverflow.com](https://stackoverflow.com/questions/15681366/importing-files-in-python)

Imports style guide

According to PEP 8, the Python style guide, you should organize imports into groups: first standard library imports, then third-party imports, and finally local application or library imports. This helps make your code more readable and maintainable.

[realpython.com](https://realpython.com/python-import/)

These are some of the most common ways to import files and packages in Python. There are more advanced techniques and customization options available in the Python import system, but these methods should cover most use cases.

**Review by alirzaMhd — CHANGES_REQUESTED** [review]

File: sign_up_window.py (hunk 2 above)
Comment (p0u74) [reply sign_up_window.py]: oh thanks it's helped

**Review by p0u74 — COMMENTED** [review]
thanks, changes submitted and main.py created.

**Pull request merged**